x-pack/plugins/enterprise_search/server/lib/enterprise_search_request_handler.ts

```diff
@@ -280,7 +281,9 @@ export class EnterpriseSearchRequestHandler {
     this.log.error(errorMessage);
```

Comment:
Quick note - these error messages coming from `server/` aren't i18n'd 🤔 I'm not sure if we have access to `i18n` from the server - I'll look into this and address in a follow-up PR if so

Reply:
bah github is being annoying and not showing all the strings. Here's an example:

https://github.com/elastic/kibana/blob/81a277a47f77ff8414b0ae90f28a0c7340384e86/x-pack/plugins/enterprise_search/server/lib/enterprise_search_request_handler.ts#L255-L257

Reply:
For context also: I was previously told server logs don't need to be i18n'd, but since we're now exposing the above string to the client-side it may be a requirement

Reply:
> // Don't expose server errors to the front-end, as they may contain sensitive stack traces 

👀 

I'm guessing showing just the message is okay?

Reply:
Right yup - we're logging showing the actual response Enterprise Search is giving us to the server (which only admins can access), but only showing this sanitized 'internal server error' to the frontend to general users

Reply:
Okay!